core/scss/components/brand-bar/_brand-bar--bright.scss

```diff
@@ -0,0 +1,5 @@
+@charset "UTF-8";
+
+#su-brand-bar.su-brand-bar--bright {
```

Comment:
I, too, would like to see the ID removed, if possible.

Reply:
Yup, only reason I did that was because I defined the default styles inside the #su-brand-bar {}, so the variants need that id selector in front to override those settings. I think I'll restructure it so that the default styles are defined in the default class, and not inside the ID {}.

And I'll add comments on the variant files.

Reply:
Looks good now!